SPECIFICATION.md

```diff
@@ -158,6 +159,73 @@ be on their own line.
 references are allowed. All references glyphs must have been previously defined.
 
+#### Whitespaces
+
+Some glyphs need additional whitespace at the right of the glyph, or even
+consist of only whitespace (no polylines). The width of this whitespace can be
+specified for every glyph with the "S" character (for "space"), followed by a
+width in the same unit as polylines.
+
+The most important space glyph (which every font file MUST contain) is " "
+(U+0020 SPACE). A typical font may use a width of 3.6 (40% of font height):
+
+    [0020] SPACE
+    S3.6
+
+As global letter spacing is applied to every glyph (including whitespace glyphs),
+the total whitespace is the addition of both values (e.g. 3.6 + 1.8 = 5.4).
+
+Whitespace definitions are also inherited when referencing other glyphs (with
+"@", see above). If a glyph has multiple whitespace definitions (either direct
+or via references), only the last definition will be used by the parser.
+
+
+## Letter Spacing / Kerning
+
+Generally the space between letters depends on the specific use-case and thus
+it's up to the application's font layout engine to determine the exact space. An
+application may even implement proper [kerning](https://en.wikipedia.org/wiki/Kerning)
+if desired.
+
+But because this is too complex for many use-cases, FontoBene allows to specify
+some spacing information directly in font files. This information can be used by
+simple layout engines to get nice looking texts with very low effort. There are
+three different options which allow to control letter spacing:
+
+1. **Global Letter Spacing:**
+
+   The parameter `letter_spacing` in the `[font]` section of the header defines
+   the letter spacing which should be put after every letter.
+
+2. **Left spacing of glyphs:**
+
+   Typically every glyph should start on the X-coordinate 0 (i.e. the left most
+   coordinate of every glyph is placed on X-coordinate 0). But if a specific
+   glyph requires more or less spacing at the left, the font designer can place
+   such glyphs slightly more left or right to make the left most point of the
+   glyph either slightly negative (reduce spacing at left) or slightly positive
+   (increase spacing at left).
+
+3. **Right spacing of glyphs:**
+
+   Glyphs can also adjust the spacing at the right of them by using the "S"
+   character for specifying additional spacing (see section "Whitespaces" above).
+
+The total space between two letters is calculated by the addition of these three
+parameters:
+
+1. Right spacing of first letter
+2. Global letter spacing
+3. Left spacing (left most X-coordinate) of second letter
+
+### Stroke Width
+
+Even if FontoBene is a stroke font, it doesn't specify the stroke width. It's up
+to the application to choose a suitable stroke width (typically between 0% and
+30% of the text height). For the font designer it's important to always specify
+spacing for a stroke width of 0 to make fonts interchangeable. Applications may
```

Comment:
A stroke width of 0 doesn't really make sense. Maybe use wording like "strokes without any added thickness" or something like that?